src/components/panel/demoGroups/style.global.css

```diff
@@ -0,0 +1,80 @@
+.menu-panel-container {
+  pointer-events: all;
```

Comment:
@bradrich You should keep in mind that `pointer-events: all` seems [to be limited to SVG](https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events). You should use `auto` if you want to reset it.